src/Validation.ContentScan.Core/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.Reflection;
+using System.Runtime.InteropServices;
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Validation.ContentScan.Core")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Validation.ContentScan.Core")]
+[assembly: AssemblyCopyright("Copyright ©  2021")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+[assembly: ComVisible(false)]
+[assembly: Guid("912a81f7-f0d9-43ef-a589-9a341eb2d6b6")]
```

Comment:
We're not super consistent with our assembly files, but let's go with something like:

```suggestion
// Copyright (c) .NET Foundation. All rights reserved.
// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.

﻿using System.Reflection;
using System.Runtime.InteropServices;

[assembly: AssemblyTitle("Validation.ContentScan.Core")]
[assembly: AssemblyDescription("Shared libraries for content scanning validations")]
[assembly: AssemblyCompany(".NET Foundation")]
[assembly: AssemblyProduct("Validation.ContentScan.Core")]
[assembly: AssemblyCopyright("Copyright ©  2021")]
[assembly: ComVisible(false)]
[assembly: Guid("912a81f7-f0d9-43ef-a589-9a341eb2d6b6")]
```